packages/date-time-picker/test/i18n.test.js

```diff
@@ -26,18 +26,21 @@ customElements.define(
       return {
         i18n: {
           type: Object,
-          value: () => {
-            return {
-              cancel: 'Peruuta', // For date picker
-
-              // formatTime and parseTime are needed so that time picker doesn't throw errors on init
-              formatTime,
-              parseTime,
-            };
-          },
         },
       };
     }
+
+    ready() {
+      super.ready();
+
+      this.i18n = {
```

Comment:
**question:** What is the reason behind this change?

Reply:
The test was failing at some point, but apparently I got it fixed. Reverted the change.